source/features/minimize-user-comments.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+import {appendBefore} from '../libs/dom-utils';
+import optionsStorage, {Options} from '../options-storage';
+import {getUsername} from '../libs/utils';
+import onNewComments from '../libs/on-new-comments';
+import anchorScroll from '../libs/anchor-scroll';
+
+const getLabel = (restore: boolean): string => `${restore ? 'Restore' : 'Minimize'} user’s comments`;
+
+async function getMinimizedUsers(): Promise<string[]> {
+	return (await optionsStorage.getAll() as Options)
+		.minimizedUsers
+		.split(/\s+/)
+		.filter(Boolean);
+}
+
+async function setMinimizedUsers(minimizedUsers: string[]): Promise<void> {
+	return optionsStorage.set({minimizedUsers: minimizedUsers.join(' ')});
+}
+
+function toggleComment(comment: HTMLElement, minimize: boolean): void {
+	if (comment.id.startsWith('issue-')) {
+		return;
+	}
```

Comment:
Hopefully the last change. It may be a good idea to _not_ hide comments by the OP, but currently:

- nothing happens when you _Minimize_ the user
- the button changes to ~~_Maximize_~~ _Restore_ but it still does nothing and makes no sense, because the comment isn't minimized.

So either we hide all comments anyway, except the first one (this can be done in `minimizeMutedUserComments`'s selector) or still apply some visual changes to the author's comments, e.g. `opacity: 0.5`